modules/odc-customizing-available-cluster-roles-using-developer-perspective.adoc

```diff
@@ -0,0 +1,34 @@
+// Module included in the following assemblies:
+//
+// applications/projects/working-with-projects.adoc
+
+[id="odc-customizing-available-cluster-roles-using-developer-perspective_{context}"]
+= Customizing the available cluster roles using the Developer perspective
+
+The users of a project are assigned to a cluster role based on their access control. These cluster roles are mentioned in the *Project* view > *Project access* tab > *Role* drop down. By default, these roles are *Admin*, *Edit*, and *View*. To add or edit the cluster roles for a project, you can customize the YAML code of the cluster.
+
+.Project roles
+image::odc_cluster_role.png[]
+
+.Procedure
+To customize the different cluster roles of a project:
+
+. Search for "Console" in the *Search* view > *Resources* drop down.
```

Comment:
When writing for GUI, location then action.
Suggest: In the *Search* view, use the *Resources* drop-down list to search for `Console`.
User defined items use backticks. We never use quotation marks in our docs. Please see the Asciidoc markup section in the OCP guidelines on GH: https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#quick-markup-reference